operators/argo-controller/tox.ini

```diff
@@ -0,0 +1,21 @@
+[flake8]
```

Comment:
Is this file also necessary in addition to the root-level tox.ini?

Reply:
I can move it all to the root-level tox.ini. I just thought it would be cleaner this way.

Reply:
Ah, sure. I'd probably move it to the root level `tox.ini` myself, but that's also not a strong preference.